examples/mobile/src/lib.rs

```diff
@@ -161,3 +165,18 @@ fn setup_music(asset_server: Res<AssetServer>, mut commands: Commands) {
         settings: PlaybackSettings::LOOP,
     });
 }
+
+// Pause audio when app goes into background and resume when it returns.
+// This is handled by the OS on iOS, but not on Android.
+fn handle_lifetime(
+    mut lifetime_events: EventReader<Lifetime>,
+    music_controller: Query<&AudioSink>,
+) {
+    for event in lifetime_events.read() {
+        match event {
+            Lifetime::Suspended => music_controller.single().pause(),
```

Comment:
Should we gate these only for Android builds?

Reply:
they will start to be useful on iOS with next version of winit, and they don't hurt for now